lienflash/lienflash-fe
# Tm/job details display

#### What’s this PR do?
- Reorganize some files to be properly components/containers
- Add feature that displays job details
- Add test for job details

#### Where should the reviewer start?
- JobDetails folder
- Some changes to Jobs and App

#### How should this be manually tested?
- in terminal, run npm test to see all passing tests
- in terminal, run npm start and click around to see job details

#### What are the relevant tickets?
- resolves #18 

#### Need Help/Support
- Need to figure out the back or close button on this page to take the user back to their previous page
